ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityEquationQuestionBlock.ts

```diff
@@ -0,0 +1,20 @@
+import { ActivityQuestionBlock } from './activityQuestionBlock';
+import { QuestionType } from './questionType';
+import { DecimalAnswer } from './decimalAnswer';
+
+export class ActivityEquationQuestionBlock extends ActivityQuestionBlock<DecimalAnswer> {
+    public maximumDecimalPlaces: number;
+
+    constructor() {
+        super();
+    }
+
+    public get questionType(): QuestionType {
+        return QuestionType.Equation;
+    }
+
+    public hasAnswer(): boolean {
```

Comment:
Could be safe to remove it'

Reply:
Sorry, I did not get the point. 
What do you mean, @josemanuel167 ?

`ActivityEquationQuestionBlock` class  should implement inherited abstract `hasAnswer` method 
from its parent abstract class `ActivityQuestionBlock`.